src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -52,6 +52,8 @@ public App()
 		/// <seealso cref="https://github.com/unoplatform/uno/issues/1741"/>
 		public void AssertIssue1790()
 		{
+			//TODO: Return macOS assert when ApplicationData is fixed
+#if !__MACOS__
```

Comment:
Make it a log warning instead of completely removing it.

Reply:
I have already another branch prepared which removes this conditional and fixes the ApplicationData problem 🙂 , so this is very temporary.